src/main/java/seedu/duke/Storage.java

```diff
@@ -110,6 +135,25 @@ public static void loadFromFile(InternshipList internshipList) {
                     String status = data[6];
                     String deadlines = data[7];
 
+                    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/yy");
+                    YearMonth defaultDate = YearMonth.of(1, 1);
+                    YearMonth start;
+                    YearMonth end;
+
+                    try {
```

Comment:
Perhaps you can put this portion into another method in the future, to cut down on the LOC